runtime/caml/misc.h

```diff
@@ -96,7 +96,7 @@ CAMLdeprecated_typedef(addr, char *);
 #endif
 
 /* Alignment */
-#ifdef __GNUC__
+#if defined(__GNUC__) || defined(__xlC__)
 #define CAMLalign(n) __attribute__((aligned(n)))
```

Comment:
Actually this attribute is supported by almost any more-or-less recent UNIX C compiler (2010+).

Reply:
It would be a good change - would you like to convert `configure.ac` to use [`AC_COMPILE_IFELSE`](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Running-the-Compiler.html#Running-the-Compiler). It would allow the error to be caught where it should be during `configure`